introduction_to_amazon_algorithms/jumpstart-foundation-models/text2text-fine-tuning-flan-t5.ipynb

```diff
@@ -0,0 +1,596 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    fine_tuned_model = Model(
</pre><p>Above we use <code>if</code> <code>else</code> to select the cases of small and base. We should either also do this here (to be more general than just <code>xl</code> )  or leave it out above   (to be specific to <code>xl</code> ) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3916/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
it seems that after fine-tuning the model is always pre-packed. e.g., this code works with small and base models

Reply:
yes by now folks have changed all the flan-t5 models to be prepacked.